app/views/dataset.scala.html

```diff
@@ -382,6 +382,24 @@ <h4>Statistics</h4>
     <script src="@routes.Assets.at("javascripts/deleteUtils.js")" type="text/javascript"></script>
     <script src="@routes.Assets.at("javascripts/people.js")" type="text/javascript"></script>
     <script src="@routes.Assets.at("javascripts/creators.js")" type="text/javascript"></script>
+
+    <script type="application/ld+json">{"context": { "@@vocab": "https://schema.org/"},
+		 "identifier": "@dataset.id",
+		 "name": "@dataset.name",
```

Comment:
can we wrap that in HTML as well, so I can't name my dataset `"; script("popup evil stuff");`

Reply:
The whole block is now wrapped, but still want to check it

Reply:
done in both dataset and file